src/include/btmem.h

```diff
@@ -830,14 +825,14 @@ struct __wt_ref {
  */
 #define WT_REF_IS_INTERNAL 0x01 /* Page is an internal page */
 #define WT_REF_IS_LEAF 0x02     /* Page is a leaf page */
+#define WT_REF_READING 0x04     /* Page is being read in */
     uint8_t flags;
 
 #define WT_REF_DISK 0       /* Page is on disk */
 #define WT_REF_DELETED 1    /* Page is on disk, but deleted */
 #define WT_REF_LOCKED 2     /* Page locked for exclusive access */
 #define WT_REF_MEM 3        /* Page is in cache and valid */
-#define WT_REF_READING 4    /* Page being read */
-#define WT_REF_SPLIT 5      /* Parent page split (WT_REF dead) */
+#define WT_REF_SPLIT 4      /* Parent page split (WT_REF dead) */
```

Comment:
I agree with the sentiment, but there are around 250 uses of `WT_REF_XXX` in the system, I'm hesitant to make this change.

@agorrod, thoughts?

Reply:
I think the name change makes sense, but even the three flags fields appear in ~65 places. I'm inclined to change those, rather than the state fields since they are all new (or changing)

I don't mind if that change happens in this PR or a subsequent ticket.

Reply:
I agree with Alex to make the distinction on the flags side. The `flags` field is new as of a few days ago so no one has their names/use too deeply ingrained yet.

Reply:
OK, I went with `WT_REF_FLAG_XXX`, pushed in 990e4eb99.